integration/container/exec_test.go

```diff
@@ -15,6 +17,75 @@ import (
 	"gotest.tools/skip"
 )
 
+// TestExecWithCloseStdin adds case for moby#37870 issue.
+func TestExecWithCloseStdin(t *testing.T) {
+	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.35"), "broken in earlier versions")
```

Comment:
This should now probably become 1.39 (assuming we backport the fix to Docker 18.09) or 1.40 (for master/next docker release - API version will have to be bumped)